frontend/src/components/app/App.js

```diff
@@ -117,30 +129,18 @@ const MENU_ITEMS = [
     text: "Libraries",
     key: "libraries",
   },
-  {
-    url: "/individuals",
-    icon: <TeamOutlined />,
-    text: "Individuals",
-    key: "individuals",
-  },
-  {
-    url: "/process-measurements",
-    icon: <ExperimentOutlined />,
-    text: "Protocols",
-    key: "process-measurements",
-  },
-  {
-    url: "/experiment-runs",
-    icon: <HddOutlined />,
-    text: "Experiments",
-    key: "experiment-runs",
-  },
   {
     url: "/datasets",
     icon: <FileZipOutlined />,
     text: "Datasets",
     key: "datasets",
   },
+  {
+    url: "/report",
+    icon: <FlagOutlined />,
+    text: "Report",
```

Comment:
use plural ("Reports") here ... we'll get more eventually...
We can already generate multiple reports ...